### PR TITLE
Added webrtc playback endpoints

### PR DESCRIPTION
**Pull request by 0xcadams**
Fixes https://linear.app/livepeer/issue/DX-156/add-new-endpoints-to-the-ui-for-playback

### CODE REVIEW

File: packages/api/src/controllers/playback.ts [hunk 1]
```diff
@@ -66,7 +71,13 @@ function newPlaybackInfo(
     type: "html5/application/vnd.apple.mpegurl",
     url: hlsUrl,
   });
-
+  if (webRtcUrl) {
+    playbackInfo.meta.source.push({
+      hrn: "WebRTC (VP8)",
+      type: "html5/video/vp8",
```
Comment (victorges) [inline packages/api/src/controllers/playback.ts]: Does this mean that Mist will be transcoding everything to VP8? 😮

**Review by victorges — APPROVED** [review]

File: packages/api/src/controllers/playback.ts (hunk 1 above)
Comment (0xcadams) [reply packages/api/src/controllers/playback.ts]: I am not sure what the `hrn` should be here. What would you include instead?

File: packages/api/src/controllers/playback.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { Request, Router } from "express";
 import { db } from "../store";
 import {
-  getPlaybackUrl as streamPlaybackUrl,
+  getHLSPlaybackUrl as getHLSPlaybackUrl,
+  getWebRTCPlaybackUrl as getWebRTCPlaybackUrl,
```
Comment (victorges) [inline packages/api/src/controllers/playback.ts]: Do we still need the rename?

File: packages/api/src/controllers/playback.ts [hunk 3]
```diff
@@ -35,8 +36,11 @@ type PlaybackInfo = {
     live?: 0 | 1;
     playbackPolicy?: Asset["playbackPolicy"] | Stream["playbackPolicy"];
     source: {
-      hrn: "HLS (TS)" | "MP4";
-      type: "html5/application/vnd.apple.mpegurl" | "html5/video/mp4";
+      hrn: "HLS (TS)" | "MP4" | "WebRTC (VP8)";
+      type:
+        | "html5/application/vnd.apple.mpegurl"
+        | "html5/video/mp4"
+        | "html5/video/vp8";
```
Comment (victorges) [inline packages/api/src/controllers/playback.ts]: Wish there was something more specific we could put here, but couldn't find anything myself

File: packages/api/src/controllers/playback.ts (hunk 1 above)
Comment (victorges) [reply packages/api/src/controllers/playback.ts]: Oh yeah I had just made that same comment in the other file. I don't know what else we could set for the `hrn`, seems like we're restricted to MIME types.

The confusion about VP8 is probably something else tho, assuming that you got these values like `WebRTC (VP8)` from Mist. Curious about how that video ends up in VP8 codec if it goes in/out of go-livepeer as H264/5

**Review by victorges — APPROVED** [review]
relgtm!

File: packages/api/src/controllers/playback.ts (hunk 3 above)
Comment (0xcadams) [reply packages/api/src/controllers/playback.ts]: I just changed this - does H264 make more sense? I checked and this is what mist is using

**Pull request merged**